.github/workflows/eslint.yml

```diff
@@ -13,10 +13,10 @@ jobs:
     if: github.event_name == 'push'
     steps:
       - uses: actions/checkout@v1
-      - name: Node.JS 12
+      - name: Node.JS 16
```

Comment:
If we're updating to Node 16 for eslint then we should also do the same for our image running on ECS